tests/testthat/test-label.R

```diff
@@ -0,0 +1,13 @@
+test_that("xportr_label: error when metadata is not set", {
+  df <- data.frame(
+    Subj = as.character(123, 456, 789),
+    Different = c("a", "b", "c"),
+    Val = c("1", "2", "3"),
+    Param = c("param1", "param2", "param3")
+  )
+
+  expect_error(
+    xportr_type(df),
```

Comment:
```suggestion
    xportr_label(df),
```

Just a typo! For a second I was wondering why was `covr` lying to us 😄

Reply:
🤦 Thank you!